components/html-embed/index.js

```diff
@@ -0,0 +1,37 @@
+export default class HtmlEmbed extends wp.element.Component {
+
+	static get defaultProps() {
+		return {
+			html: '',
+		};
+	}
+
+	componentDidMount() {
+		const body = this.node;
+		const { html } = this.props;
```

Comment:
Maybe just use a default value `html = ''` here instead of the defaultProps. This is generally the way we're doing this in the other components (though I think it's not consistent everywhere)